prow/apis/prowjobs/v1/types.go

```diff
@@ -71,6 +72,8 @@ const (
 	JenkinsAgent ProwJobAgent = "jenkins"
 	// KnativeBuildAgent means prow will schedule the job via a build-crd resource.
 	KnativeBuildAgent ProwJobAgent = "knative-build"
+	// TektonAgent means prow will schedule the job via a tekton PipelineRun CRD resource.
+	TektonAgent = "tekton-pipeline"
```

Comment:
Would just `tekton` be sufficient? Or do we need the `pipeline` suffix as well?

Reply:
I am fine either way.